Resources/Prototypes/Entities/Structures/Storage/Closets/Lockers/lockers.yml

```diff
@@ -32,7 +32,22 @@
       state_open: qm_open
       state_closed: qm_door
   - type: AccessReader
-    access: [["Quartermaster"]] 
+    access: [["Quartermaster"]]
+
+- type: entity
+  id: LockerSalvageSpationaut
+  parent: LockerBase
+  name: salvage spationaut's equipment
+  description: Nevermind the pickaxe.
+  components:
+  - type: Appearance
+    visuals:
+    - type: StorageVisualizer
+      state: mining
+      state_open: mining_open
+      state_closed: mining_door
+  - type: AccessReader
+    access: [["Salvage"]] 
```

Comment:
no fill?

Reply:
Entity fill is stored in Resources/Prototypes/Catalog/Fills/Lockers/cargo.yml per standard.